### PR TITLE
fix: disable jit when building on mac

### PR DESCRIPTION
**Pull request by id**

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -50,7 +37,7 @@ jobs:
         with:
           context: .
           platforms: linux/amd64,linux/arm64
-          push: ${{ startsWith(github.ref, 'refs/tags/') || inputs.publish_release_artifacts == true }}
+          push: false
```
Comment (thalesmg) [inline .github/workflows/build.yaml:40]: Q: should this workflow dump the image as an artifact now that it no longer pushes the image?

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+          fetch-depth: 0
+      - uses: docker/setup-qemu-action@v2
+      - uses: docker/setup-buildx-action@v2
+      - uses: docker/metadata-action@v4
+        id: meta
+        with:
+          images: ${{ github.repository }}
+          tags: |
+            type=ref,event=branch
+            type=ref,event=pr
```
Comment (thalesmg) [inline .github/workflows/release.yaml:39]: No longer applicable?  :scissors:

**Review by thalesmg — APPROVED** [review]

File: .github/workflows/build.yaml (hunk 1 above)
Comment (id) [reply .github/workflows/build.yaml:40]: we can add it if there is a need.

File: .github/workflows/release.yaml (hunk 2 above)
Comment (id) [reply .github/workflows/release.yaml:39]: yeah, I'll remove it in the next pr

**Pull request merged**